### PR TITLE
Pie 3130 upgrade Ant Design version

### PR DESCRIPTION
**Pull request by MartinM1312**
## 💅🏼 What issue does this fix?
Ant Design version upgraded

## 🍂 Before You Submit

* [x] Did you write & run tests and linters?
* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment

* [ ] Migrations
* [ ] Dependencies

### CODE REVIEW

File: client/src/Dashboard/DatePicker.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import dayjsGenerateConfig from 'rc-picker/lib/generate/dayjs'
 import generatePicker from 'antd/lib/date-picker/generatePicker'
-import 'antd/lib/date-picker/style/index'
+
+// import 'antd/lib/date-picker/style/index'
```
Comment (fabianraf) [inline client/src/Dashboard/DatePicker.js:4]: If not needed, remove it.

File: client/src/_shared/Header.js [hunk 2]
```diff
@@ -90,6 +91,7 @@ export function Header() {
                 className={
                   location.pathname.includes('/attendance') ? 'underline' : ''
                 }
+                id={'dashboard-tab'}
```
Comment (fabianraf) [inline client/src/_shared/Header.js:94]: You cannot have the same id twice in a form

**Review by fabianraf — CHANGES_REQUESTED** [review]

File: client/src/Dashboard/DatePicker.js (hunk 1 above)
Comment (MartinM1312) [reply client/src/Dashboard/DatePicker.js]: done

File: client/src/_shared/Header.js (hunk 2 above)
Comment (MartinM1312) [reply client/src/_shared/Header.js]: fixed

**Pull request merged**